sql/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import io.crate.metadata.view.ViewsMetaData;
+import org.elasticsearch.cluster.service.ClusterService;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.is;
+
+public class ViewsITest extends SQLTransportIntegrationTest {
+
+    @Test
+    public void testViewCanBeCreated() throws Exception {
+        execute("create view v1 as select 1");
+        for (ClusterService clusterService : internalCluster().getInstances(ClusterService.class)) {
+            ViewsMetaData views = clusterService.state().metaData().custom(ViewsMetaData.TYPE);
+            assertThat(views, Matchers.notNullValue());
+            assertThat(views.contains(sqlExecutor.getDefaultSchema() + ".v1"), is(true));
+        }
+    }
+
+    @Test
+    public void testViewCanBeCreatedAndThenReplaced() throws Exception {
+        execute("create view v2 as select 1");
+        execute("create or replace view v2 as select 1");
```

Comment:
Shouldn't we use another query? for the 2nd (replace) statement

Reply:
The tests generally don't verify the saved query definition yet. Will extend this once we've the information schema available

Reply:
ok, thought we can check it from ViewsMetaData but it's ok if it's gonna be replaced with "full-blown" integ test on information_schema later on.